test/unit/resources/aws_billing_report_test.rb

```diff
@@ -1,5 +1,11 @@
 require 'helper'
-require_relative 'aws_billing_backend'
+require 'inspec/resource'
+require 'resources/aws/aws_billing_report'
+
+require_relative 'aws_billing_backend' # TODO: move this to mocks
```

Comment:
Note - as the AWS resources are being moved to the inspec-resource pack, development has stopped on these resources - and the resource pack has a different test harness, IIRC. So, we don't need to invest too much here.